File: src/AppRegistryService.Contract/Models/AppRegistryServiceError.cs
namespace AppRegistryService.Contract.Models;

/// <summary>
/// Defines an AppRegistry service error.
/// </summary>
public sealed class AppRegistryServiceError
{
    /// <summary>
    /// Error code.
    /// </summary>
    public WellKnownAppRegistryServiceErrorCode ErrorCode { get; set; }
}

File: web/src/models/AppRegistryServiceError.ts
import WellKnownAppRegistryServiceErrorCode from './WellKnownAppRegistryServiceErrorCode';

/**
 * Defines an AppRegistry service error.
 */
export interface AppRegistryServiceError {
    /**
     * Error code.
     */
    errorCode: WellKnownAppRegistryServiceErrorCode;
}